src/uucore/src/lib/features/fs.rs

```diff
@@ -743,6 +743,52 @@ pub fn path_ends_with_terminator(path: &Path) -> bool {
         .map_or(false, |wide| wide == b'/'.into() || wide == b'\\'.into())
 }
 
+pub mod sane_blksize {
+
+    #[cfg(not(target_os = "windows"))]
+    use std::os::unix::fs::MetadataExt;
+    use std::{fs::metadata, path::Path};
+
+    pub const DEFAULT: u64 = 512;
+    pub const MAX: u64 = (u32::MAX / 8 + 1) as u64;
+
+    /// Provides sanity checked blksize value from the provided value.
+    /// If the provided value is a invalid values a meaningful adaption
+    /// of that value is done.
+    pub fn sane_blksize(st_blksize: u64) -> u64 {
```

Comment:
Why is this public API?

Another very small nitpick: I think the docstring for these functions should be

```
/// Provides sanity checked blksize value from the provided value.
///
/// If the provided value is a invalid values a meaningful adaption
/// of that value is done.
```

That is a format that rustdoc understands where the first line is the summary and the rest is a more expansive description.

Reply:
You are asking because this function is only used by `sane_blksize_from_metadata` and `sane_blksize_from_path` which are in the same module.
So its not needed to have the function `sane_blksize` public.
Right?

I would see the three function on the same visibility level. Two below are just convenience wrappers.
Is there any big concern from your side why not making the function `sane_blksize` public?

I adapted the comments to follow the rustdoc format. Thanks for the hint.

Reply:
Yes, that's indeed what I meant. I just think it's good practice to only expose what we actually need.